ref/net46/Microsoft.Build.Utilities.Core/Microsoft.Build.Utilities.Core.cs

```diff
@@ -317,6 +317,7 @@ public enum TargetDotNetFrameworkVersion
         Version461 = 8,
         Version462 = 10,
         Version47 = 11,
+        Version471 = 12,
```

Comment:
Is it OK for VersionLatest to point to 462?

Reply:
Yeah, since it's only an update changing `VersionLatest` could be a breaking change. That's why I added `Latest` last time so that consumers could target the appropriate flag.